ddtrace/internal/debug.py

```diff
@@ -137,3 +137,82 @@ def collect(tracer):
         partial_flush_enabled=tracer._partial_flush_enabled,
         partial_flush_min_spans=tracer._partial_flush_min_spans,
     )
+
+
+def pretty_collect(tracer):
+    class bcolors:
+        HEADER = "\033[95m"
+        OKBLUE = "\033[94m"
+        OKCYAN = "\033[96m"
+        OKGREEN = "\033[92m"
+        WARNING = "\033[93m"
+        FAIL = "\033[91m"
+        ENDC = "\033[0m"
+        BOLD = "\033[1m"
+
+    info = collect(tracer)
+
+    info_pretty = """{0}{1}Tracer Configurations:{2}
+    Tracer enabled: {3}
+    Debug logging: {4}
+    Writing traces to: {5}
+    Agent error: {6}
+    App Analytics enabled(deprecated): {7}
+    Log injection enabled: {8}
+    Health metrics enabled: {9}
+    Priority sampling enabled: {10}
+    Partial flushing enabled: {11}
+    Partial flush minimum number of spans: {12}
+    {13}{14}Tagging:{15}
+    DD Service: {16}
+    DD Env: {17}
+    DD Version: {18}
+    Global Tags: {19}
+    Tracer Tags: {20}""".format(
+        bcolors.OKBLUE,
+        bcolors.BOLD,
+        bcolors.ENDC,
+        info.get("tracer_enabled"),
+        info.get("debug"),
+        info.get("agent_url") or "Not writing at the moment, is your tracer running?",
+        info.get("agent_error") or "None",
+        info.get("analytics_enabled"),
+        info.get("log_injection_enabled"),
+        info.get("health_metrics_enabled"),
+        info.get("priority_sampling_enabled"),
+        info.get("partial_flush_enabled"),
+        info.get("partial_flush_min_spans") or "Not set",
+        bcolors.OKGREEN,
+        bcolors.BOLD,
+        bcolors.ENDC,
+        info.get("service") or "None",
+        info.get("env") or "None",
+        info.get("dd_version") or "None",
+        info.get("global_tags") or "None",
+        info.get("tracer_tags") or "None",
+    )
+
+    summary = "{0}{1}Summary{2}".format(bcolors.OKCYAN, bcolors.BOLD, bcolors.ENDC)
+
+    if info.get("agent_error"):
+        summary += "\n\n{0}ERROR: It looks like you have an agent error: '{1}'\nThe most common error is a connection error. If you're experiencing a connection error, please make sure you've followed the setup for your particular environment so that the tracer and Datadog agent are configured properly to connect, and that the Datadog agent is running: https://docs.datadoghq.com/tracing/setup_overview/setup/python/?tab=containers#configure-the-datadog-agent-for-apm\nIf your issue is not a connection error then please reach out to support for further assistance: https://docs.datadoghq.com/help/{2}".format(
```

Comment:
I wonder if we should shorten these, a little lengthy to read. If we have a link anyways?

Reply:
I think that it's nice for customers to have a bit of understanding about why they're setting the tags. The docs link to how to set the tags, but don't really explain why. I guess customer's can look into it further themselves though. I've shortened the messages.

Reply:
yeah, another reason why if we move _all_ of these to our own troubleshooting docs then we can control the messaging to make sure it is what we want them to be rather than relying on what corp docs say.

Reply:
the "link to our own docs" bit is just a mild suggestion, not a blocker... especially if others think it is crazy idea